crypto/x509/x509_lu.c

```diff
@@ -166,6 +171,7 @@ X509_STORE *X509_STORE_new(void)
 
 err:
     X509_VERIFY_PARAM_free(ret->param);
+    X509err(X509_F_X509_STORE_NEW, ERR_R_MALLOC_FAILURE);
```

Comment:
... instead of here.

Reply:
IMHO the code following the error labels should only do cleanup and not issue the error message. This might be convenient because one does not have to repeat the error call, but when debugging a failure it is easier to locate the cause if it immediately precedes the error call.